4 reports.sql

```diff
@@ -0,0 +1,26 @@
+--1) Show a list of all soldiers with all details sorted by IQ level from high to low.
+select *
+from Soldiers s
+order by s.IQlevel desc
+
+--2) I need a list of all soldiers sorted by age at enlistment from high to low. Do not include columns that are not relevant to this list.
+select s.DOB, s.DateOfEnlistment, Age = datediff(year,s.DOB,s.DateOfEnlistment)
```

Comment:
Show the names etc. The things not relevant are IQ level, place of residence, unit etc.

Reply:
Added relevant columns